platform/vcs-impl/src/com/intellij/openapi/vcs/actions/AnnotateToggleAction.java

```diff
@@ -204,7 +204,11 @@ public void dispose() {
       gutters.add(mergeSourceGutter);
     }
 
-    final LineAnnotationAspect[] aspects = fileAnnotation.getAspects();
+    final List<LineAnnotationAspect> aspects = new ArrayList<>();
+    aspects.addAll(Arrays.asList(fileAnnotation.getAspects()));
+    for (AnnotationGutterColumnProvider extension : AnnotationGutterColumnProvider.EP_NAME.getExtensions()) {
+      aspects.add(extension.createColumn(fileAnnotation));
```

Comment:
`ContainerUtil.addIfNotNull`, to avoid NPE?

`createColumn` is `@Nullable`, as some extension might need `GitFileAnnotation` and can't handle `HgFileAnnotation`.